### ISSUE TITLE
Personal blog: add comments section

### ISSUE THREAD

**Issue by 4unkur** [intelliants/subrion#127]

**Comment by glebovsky** [comment]
Lets discuss this? 

Native comments, or some sort of integration like Disqus or similar?

**Comment by vbezruchkin** [comment]
native subrion comments, so they could be linked to pages or plugin generated content, for example.

**Comment by glebovsky** [comment]
You mean not only for blog, right?

**Comment by vbezruchkin** [comment]
Yep, comments might be useful for events as well. Or any other stuff that could be generated by plugins - news

**Comment by glebovsky** [comment]
> Yep, comments might be useful for events as well. Or any other stuff that could be generated by plugins - news

Thats great! You should rename this issue then.

**Comment by vbezruchkin** [comment]
I guess @4unkur needs to think wide open then ;)

**Comment by 4unkur** [comment]
Then it'll be another plugin "Comments" but for plugins, not for packages as we already have such plugin http://www.subrion.org/plugin/comments.html

**Comment by vbezruchkin** [comment]
Subrion 4.2.x allows plugins to add items, so it's possible to implement this feature in our Comments plugin.

It will be possible once we migrate content plugins to items.
